packages/xrpl/src/models/methods/depositAuthorized.ts

```diff
@@ -15,6 +15,12 @@ export interface DepositAuthorizedRequest
   source_account: string
   /** The recipient of a possible payment. */
   destination_account: string
+  /**
+   * The object IDs of Credential objects. If this field is included, then the
+   * credential will be taken into account when analyzing whether the sender can send
+   * funds to the destination.
+   */
+  credentials?: string[]
 }
 
 /**
```

Comment:
The response type also needs to be updated (I think the latest version of the spec has this in https://github.com/XRPLF/XRPL-Standards/pull/250

Reply:
thanks, added